RepoDb.Core/RepoDb/Reflection/Compiler/Compiler.cs

```diff
@@ -1,15 +1,15 @@
+using RepoDb.Enumerations;
+using RepoDb.Exceptions;
+using RepoDb.Extensions;
+using RepoDb.Interfaces;
+using RepoDb.Resolvers;
 using System;
 using System.Collections.Generic;
 using System.Data;
 using System.Data.Common;
 using System.Linq;
 using System.Linq.Expressions;
 using System.Reflection;
-using RepoDb.Enumerations;
-using RepoDb.Exceptions;
-using RepoDb.Extensions;
-using RepoDb.Interfaces;
-using RepoDb.Resolvers;
```

Comment:
Auto-ordering by Visual Studio. Perhaps an .editorconfig should be added to the repository to avoid such changes in the future.